### ISSUE TITLE
MobyLinuxVM failed to realize - fix

### ISSUE THREAD

**Issue by jeffstokes72** [docker/for-win#788]
### Expected behavior
Install Docker and have it start without error

### Actual behavior
MobyLinuxVM failed to realize

### Information

  - Diagnostic ID from "Diagnose & Feedback" in the menu.
**A556701F-1849-4397-BBE6-C4288D5C28AA/2017-06-02_03-30-47**
  - a reproducible case if this is a bug, Dockerfiles FTW
  - page URL if this is a docs issue or the name of a man page
  - host distribution and version (Windows version, build number, etc)
**Win 10.0.16199.1000**

### Steps to reproduce the behavior

  1. install Docker (stable or edge)
  2. fail to launch Docker

### 
I fixed this issue. It is documented here.
https://illuminati.services/2017/06/02/docker-on-windows-mobylinuxvm-failed-to-realize-fixed/

TL;DR hyper-v kept stale info with invalid paths to the VM storage area (registry, binary blob somewhere, who knows), Docker install picked them up and didn't pre-flight check them (presumably)

**Comment by jasonbivins** [comment]
Hi @jeffstokes72  thanks for posting!

**Comment by jasonbivins** [comment]
Hi @jeffstokes72 

I captured this as a KB - I'll go ahead and close it now.  Thanks again!

**Comment by judge2020** [comment]
Would like to note that it would be nice to provide a more descriptive message when this occurs, possibly run a pre-check to see if the Hyper-V folder exists on the system and warn if it isn't.

**Comment by DanTup** [comment]
This issue has a label of "status/3-fixed", but I just hit exactly the same issue today. I'd also deleted the Hyper-V folder after using it in the past, and Docker crashed with the same error.

**Comment by wrstlrlp13** [comment]
Can't believe Hyper-V doesn't realize those are wrong... Thanks!!!

**Comment by jeffstokes72** [comment]
> This issue has a label of "status/3-fixed", but I just hit exactly the same issue today. I'd also deleted the Hyper-V folder after using it in the past, and Docker crashed with the same error.

did you get the directory hyper-v thinks it's using for the store? It can be changed in the hyper-v server config. Sorry for my late reply.
